### PR TITLE
chore: remove the code related to js prepare as webpack overrides prepareJS method with own logic

### PR DESCRIPTION
**Pull request by Fatme**
## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

## What is the new behavior?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

File: lib/services/plugins-service.ts [hunk 1]
```diff
@@ -108,38 +105,6 @@ export class PluginsService implements IPluginsService {
...
 
-	public async prepare(dependencyData: IDependencyData, platform: string, projectData: IProjectData, projectFilesConfig: IProjectFilesConfig): Promise<void> {
-		platform = platform.toLowerCase();
-		const platformData = this.$platformsData.getPlatformData(platform, projectData);
-		const pluginData = this.convertToPluginData(dependencyData, projectData.projectDir);
-
-		const appFolderExists = this.$fs.exists(path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME));
-		if (appFolderExists) {
-			this.preparePluginScripts(pluginData, platform, projectData, projectFilesConfig);
-			await this.preparePluginNativeCode(pluginData, platform, projectData);
```
Comment (KristianDD) [inline lib/services/plugins-service.ts]: This should be moved somewhere maybe.

File: lib/services/prepare-platform-js-service.ts [hunk 2]
```diff
@@ -36,23 +34,7 @@ export class PreparePlatformJSService extends PreparePlatformService implements
 	@performanceLog()
 	@hook('prepareJSApp')
 	public async preparePlatform(config: IPreparePlatformJSInfo): Promise<void> {
-		if (!config.changesInfo || config.changesInfo.appFilesChanged || config.changesInfo.changesRequirePrepare) {
-			await this.copyAppFiles(config);
-			this.copyAppResourcesFiles(config);
```
Comment (KristianDD) [inline lib/services/prepare-platform-js-service.ts]: We might need to move this call somewhere else.

File: lib/services/prepare-platform-js-service.ts [hunk 3]
```diff
@@ -36,23 +34,7 @@ export class PreparePlatformJSService extends PreparePlatformService implements
...
-		}
-
-		if (config.changesInfo && !config.changesInfo.changesRequirePrepare) {
-			// remove the App_Resources folder from the app/assets as here we're applying other files changes.
-			const appDestinationDirectoryPath = path.join(config.platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME);
-			const appResourcesDirectoryPath = path.join(appDestinationDirectoryPath, path.basename(config.projectData.appResourcesDirectoryPath));
-			if (this.$fs.exists(appResourcesDirectoryPath)) {
-				this.$fs.deleteDirectory(appResourcesDirectoryPath);
```
Comment (KristianDD) [inline lib/services/prepare-platform-js-service.ts]: We could use this clean up too.

**Comment by Fatme** [comment]
We discussed and agreed on all comments in private conversation.

**Pull request merged**